RFS/src/main/java/com/rfs/cms/OpenSearchCmsEntry.java

```diff
@@ -44,23 +41,26 @@ public ObjectNode toJson() {
             node.put(FIELD_STATUS, status.toString());
             return node;
         }
+
+        @Override
+        public String toString() {
+            return this.toJson().toString();
+        }
```

Comment:
Make this a method on the base class on CmsEntry.Snapshot or higher that defaults to json - otherwise you are repeating this boiler plate code and if you miss an override you won't know until you are dealing with a log line `<Snapshot@1234221>`

Reply:
Done